app/components/inplace-boolean.js

```diff
@@ -1,14 +1,19 @@
 import Ember from 'ember';
-import InPlace from 'ilios/mixins/inplace';
+import InPlaceValidation from 'ilios/mixins/inplace-validation';
```

Comment:
What does validation of a boolean do?  I guess we could prevent it from being false, but that seems like it wouldn't ever happen.